packages/xcoobee-cookie-kit-web/src/index.js

```diff
@@ -114,6 +114,12 @@ class CookitKitInitializer {
     document.body.appendChild(managedCookie);
   }
 
+  display(value) {
```

Comment:
This needs to be documented some where the end user (the developer) can find it. Somewhere in `packages/xcoobee-cookie-kit-web/README.md` looks to be the right location. Including an example too would be good.

Reply:
Bilal said that we don't need to update a ReadMe file for know for some specific reasons.

Reply:
Okay.